soh/soh/Enhancements/controls/GameControlEditor.cpp

```diff
@@ -315,6 +315,7 @@ namespace GameControlEditor {
             UIWidgets::Spacer(5);
              Ship::BeginGroupPanel("Walk Modifier", ImGui::GetContentRegionAvail());
             UIWidgets::PaddedEnhancementCheckbox("Toggle modifier instead of holding", "gWalkSpeedToggle", true, false);
+            UIWidgets::PaddedEnhancementCheckbox("Don't affect jump distance/velocity", "gWalkModifierNoVelocity", true, false);
```

Comment:
`gWalkModifierNoVelocity` feels like an odd name, maybe something like `gWalkModifierChangesJump`?

Reply:
Yeah I was conflicted because I figure the new option should be off by default and the original behavior of the speed modifier (which changes jump) should be preserved. But that was just me being complicated

Reply:
maybe `gWalkModifierDoesntChangeJump` then? My issue is just that "No Velocity" doesn't mention jumping at all